src/shared/canvas/DataExplorerGraphFlow/DateExplorerGraphFlow.spec.tsx

```diff
@@ -124,12 +124,13 @@ describe('DataExplorerGraphFlow', () => {
     );
     history.push('/another-page');
     const dataExplorerState = store.getState().dataExplorer;
-    const localstorage = localStorage.getItem(DATA_EXPLORER_GRAPH_FLOW_DIGEST);
+    const localstorage = sessionStorage.getItem(
```

Comment:
small thing: maybe we should rename the variable to `sessionStorageItem` instead of `localStorage`